backend/app/project/routes.py

```diff
@@ -1,8 +1,61 @@
-import uuid
 from flask import jsonify, request, abort
 from app import db
-from app.models import Project
 from . import project
+from app.models import Project
+
+
+# get all endpoints
+@project.route("", methods=["GET"])
```

Comment:
This is the same method as that's created in #98 ?

Reply:
Yes sorry shouldnt have pulled from the #98 branch